app/controllers/api/v1x2/order_processes_controller.rb

```diff
@@ -55,6 +75,16 @@ def update_association(association)
           order_process, params.require(:portfolio_item_id), association
         ).process.order_process
       end
+
+      def for_resource_object?
+        raise ::Catalog::InvalidParameter, "Invalid resource object params: #{resource_params}" unless resource_params.length.zero? || resource_params.length == 3
+
+        !!(resource_params[:app_name] && resource_params[:object_id] && resource_params[:object_type])
+      end
+
+      def resource_params
+        params.permit(:object_type, :object_id, :app_name).to_h
+      end
```

Comment:
These should probably be private methods, ya? Or, possibly the logic should be pushed down into the service.

Reply:
Changed to private methods.